packages/frontend/app/components/reports/curriculum/learner-groups.hbs

```diff
@@ -31,6 +35,9 @@
       {{#if this.reportRunning}}
         {{#each (sort-by "courseTitle" @courses) as |c|}}
           <tr>
+            {{#if this.hasMultipleSchools}}
+              <td>{{11}}</td>
```

Comment:
This should be the name of the school for this course. Will required doing a local getter for `@courses` and then augmenting it with the school data, but since this can be looked up without async loading it makes more sense to have a real name here and it will space out better so when the load finished there won't be a re-flow of the table.